src/test/java/org/jitsi/jicofo/OSGiHandler.java

```diff
@@ -72,6 +73,12 @@ public MockXmppProvider getXmppProvider()
 
     public void shutdown()
     {
+        System.clearProperty("org.jitsi.jicofo.PING_INTERVAL");
+        System.clearProperty(XmppClientConnectionConfig.legacyXmppDomainPropertyName);
+        System.clearProperty(XmppClientConnectionConfig.legacyDomainPropertyName);
+        System.clearProperty(XmppClientConnectionConfig.legacyUsernamePropertyName);
+        JitsiConfig.Companion.reloadNewConfig();
+
```

Comment:
This doesn't logically seem part of the "Move XmppTest" commit - what's it doing?

Reply:
Moving the test changed the order in which tests are run, which exposed a bug with some other tests changing the configuration. Now that I look at it more carefully (though I still don't understand it fully), it should probably be JicofoConfigTest that needs to clean-up after itself.

Reply:
Aha. It used to be that `OSGiHandler.init` never ran before `XmppConfigTest`. But now `AllocateConferenceTest` runs prior to `XmppConfigTest`, and if `OSGiHandler` leaves these properties behind it brakes things.